templates/broadcom_icos_show_version.textfsm

```diff
@@ -0,0 +1,41 @@
+Value System_Description (.+)
+Value Machine_Type (.+)
+Value Machine_Model (.+)
+Value Serial_Number (.+)

```

Comment:
Could you also look at other `show_version` from other platforms and make sure you're using the same named values as those? This helps bring our templates more vendor agnostic in the way they return the data. Users can act on the same keys for other platforms.

Reply:
Went thru several show versions and took a hint on regularly used named values. A recent Ubiquity pull showed very similar patterns so used as a guide, mainly: https://github.com/networktocode/ntc-templates/tree/master/tests/ubiquiti_edgeswitch/show_version but also other common names were replaced. No references were found for the rest so, left as is or shortened if the name made sense still.